ci-scripts/yocto.groovy

```diff
@@ -228,7 +228,8 @@ void buildManifest(String variantName, String imageName, boolean smokeTests=fals
             archiveCache(yoctoDir, archive, env.YOCTO_CACHE_ARCHIVE_PATH)
             // If nightly build, we store the artifacts as well
             boolean nightly = env.NIGHTLY_BUILD == "true"
-            if (nightly) {
+            boolean weekly = env.WEEKLY_BUILD == "true"
+            if (nightly || weekly) {
```

Comment:
Based on this, the arp image will be archived once a week, while all the other images will be archived 5 times a week. Are we okay with this?

Reply:
My reasoning here is that for now this could be ok. ARP isn't updated very often and we can change which variants are included in the Nightly/Weekly very easily. ARP was put in the weekly originally since it is not updated very often, but at this moment we aren't including any other variants, but I think we should.